lib/common/client/concerns/mhv_session_based_client.rb

```diff
@@ -45,7 +46,9 @@ def authenticate
       # @return [Sm::ClientSession] if a SM (Secure Messaging) client session
       #
       def get_session
-        env = perform(:get, 'session', nil, auth_headers)
+        env = set_raven_tag(:error, 'mhv_session') do
```

Comment:
Can you find of a method name that captures what's really going on here? It's not setting the tag in the way a casual reading would indicate (esp since the corresponding method literally deletes the tag.